### PR TITLE
Make setup-keeper action less flaky

### PR DESCRIPTION
**Pull request by jaredly**
## Summary:
For some inscrutable reason, using the bash HEREDOC stuff worked a couple of times, and then never again. I trust javascript more anyway, so I'm happy with this change.

## Test plan:
see the check working!

### CODE REVIEW

File: actions/setup-keeper/action.yml [hunk 1]
```diff
@@ -11,14 +11,16 @@ runs:
       shell: bash
       run: pip3 install cryptography==37.0.2 pyOpenSSL==22.0.0 keepercommander==16.6.5
 
+    - uses: actions/github-script@v6
+      with:
+        script:
+          const config = `${{ inputs.config }}`;
```
Comment (jeresig) [inline actions/setup-keeper/action.yml:17]: I'm surprised there isn't a way to access incoming inputs as variables in `github-script`?

File: actions/setup-keeper/action.yml [hunk 2]
```diff
@@ -11,14 +11,16 @@ runs:
...
-        cat << EOF > $HOME/.keeper-config.json
-        ${{ inputs.config }}
-        EOF
-        echo "KEEPER_CONFIG_FILE=$HOME/.keeper-config.json" >> $GITHUB_ENV
+      run: echo "KEEPER_CONFIG_FILE=$HOME/.keeper-config.json" >> $GITHUB_ENV
 
     - name: Check that keeper is installed correctly
       shell: bash
-      run: keeper whoami | grep '@khanacademy.org'
+      run: keeper this-device
```
Comment (jeresig) [inline actions/setup-keeper/action.yml:26]: What is this change doing?

**Review by jeresig — APPROVED** [review]
Huh - I wonder why it was failing with the other technique?

File: actions/setup-keeper/action.yml (hunk 2 above)
Comment (jaredly) [reply actions/setup-keeper/action.yml:26]: making the consistency check more generic (not relying on the ka.org namespace). Either command will fail if authentication didn't happen correctly.

File: actions/setup-keeper/action.yml (hunk 1 above)
Comment (jaredly) [reply actions/setup-keeper/action.yml:17]: yeah I was also surprised, but composite actions don't get the ENV variables made for inputs automatically. I guess it makes usage more explicit?

Comment (jeresig) [reply actions/setup-keeper/action.yml:17]: I wonder if you could use this: https://github.com/actions/github-script#use-env-as-input

so define an ENV for the config for this step and then access it via `process.env.KEEPER_CONFIG`.

**Pull request merged**